### PR TITLE
Make pg_cron compile for Windows using Visual Studio 

### PR DESCRIPTION
**Pull request by Godwottery**
uint is as far as I can tell not a standard data-type. Visual Studio does not like it, and will not compile the code. I think that unsigned int should be asuitable replacement. 

This is preparation for additional work to get pg_cron running on Windows.

### CODE REVIEW

File: include/task_states.h [hunk 1]
```diff
@@ -34,7 +34,7 @@ typedef struct CronTask
 	int64 jobId;
 	int64 runId;
 	CronTaskState state;
-	uint pendingRunCount;
+	unsigned int pendingRunCount;
```
Comment (marcocitus) [inline include/task_states.h]: I think this is meant to be `uint32` (a type defined by postgres), but fine with this change.

File: include/job_metadata.h [hunk 2]
```diff
@@ -35,10 +35,10 @@ extern bool CronJobCacheValid;
...
 
 /* functions for retrieving job metadata */
-extern void InitializeJobMetadataCache(void);
-extern void ResetJobMetadataCache(void);
-extern List * LoadCronJobList(void);
-extern CronJob * GetCronJob(int64 jobId);
+PGDLLEXPORT extern void InitializeJobMetadataCache(void);
+PGDLLEXPORT extern void ResetJobMetadataCache(void);
+PGDLLEXPORT extern List * LoadCronJobList(void);
+PGDLLEXPORT extern CronJob * GetCronJob(int64 jobId);
```
Comment (marcocitus) [inline include/job_metadata.h]: Normally PGDLLEXPORT is only necessary when functions need to be called from another library/extension. Is that the case here?

Comment (Godwottery) [reply include/job_metadata.h]: I am checking to see if it will work without.

**Comment by Godwottery** [comment]
Reference #22

**Comment by Godwottery** [comment]
Does this PR have a chance of being merged if I resubmit?

**Comment by marcianobandeira** [comment]
Is it currently possible to install pg_cron on Windows?

**Pull request closed without merging**